File: python/src/business.py
class Business:
    def __init__(self, business_name):
        # TODO: Implement constructor
        self.business_name = business_name

    def get_business_name(self):
        # TODO: Implement getter
        return self.business_name

    def set_business_name(self, business_name):
        # TODO: Implement setter
        if business_name == None or business_name == "":
            print("Invalid Business Name.")
        else:
            self.business_name = business_name

File: src/main/java/io/zipcoder/Business.java
package io.zipcoder;

public class Business {
    private String businessName;

    public Business(String businessName) {
        // TODO: Implement constructor
        this.businessName = businessName;
    }

    public String getBusinessName() {
        // TODO: Implement getter
        return businessName;
    }

    public void setBusinessName(String businessName) {
        // TODO: Implement setter
        if(businessName != null && !businessName.isEmpty()) {
            this.businessName = businessName;
        } else {
            System.out.println("Invalid input");
        }
    }
}
